lib/mayaUsd/render/vp2RenderDelegate/meshViewportCompute.cpp

```diff
@@ -1303,6 +1304,11 @@ bool MeshViewportCompute::execute(
 
     prepareAdjacencyBuffer();
 
+    if (!hasOpenGL()) {
```

Comment:
Fix the expression, and move the initialization code before any OpenGL calls are made. I think in earlier versions of USD OpenGL was initialized for us, and so the old code never did anything useful. Now that OpenGL is not initialized it's obvious the old version was wrong.